tools/fv_nudge.F90

```diff
@@ -3545,16 +3545,17 @@ subroutine pmaxmin( qname, a, imax, jmax, fac )
       class(*) a(imax,jmax)
```

Comment:
What is "class(*)"? Is this new Fortran functionality, and if so what version?

Reply:
It is F2003. These changes are part of [the efforts](https://github.com/NOAA-GFDL/FMS/pull/857) to allow FMS to handle both 32- and 64-bit interfaces.